next-app/app/components/navbar/navbar.js

```diff
@@ -1,20 +1,29 @@
 'use client';
-import Image from 'next/image';
 import { usePathname } from 'next/navigation';
-import { Nav, Navbar, NavbarToggle } from 'react-bootstrap';
+import { Image, Nav, Navbar, NavbarToggle } from 'react-bootstrap';
 import Container from 'react-bootstrap/Container';
 
 function Navibar() {
+  let page = {
+    '/': 'Home',
+    '/schedule': 'Schedule',
+    '/score': 'Scorekeeper',
+    '/matches': 'Match History',
+    '/players': 'Players',
+  };
+
   return (
     <Navbar expand="lg" className="bg-body-tertiary">
       <Container>
         <Navbar.Brand href="/">
           <Image alt="" src="/logo.png" style={{ borderRadius: 7 }} width="48" height="48"></Image>
         </Navbar.Brand>
+        <h1 className="d-lg-none display-3">{page[usePathname()]}</h1>
```

Comment:
i think there may be some mobile / desktop breakpoint mismatches somewhere. when my width is 1150px, i dont get the page name in the page or the nav bar